changelog/19111.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+Fix a bug that did not allow WAL rollback to handle partial failures when creating API keys
```

Comment:
```suggestion
database/mongodb-atlas: Fix a bug that did not allow WAL rollback to handle partial failures when creating API keys
```
I think the changelog usually prefixes the component targeted by the change?

Reply:
thats right! Thanks, nice catch